packages/CMS/.babelrc

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
We should not add a additional `.babelrc` in every packages. All package's `.babelrc` should extend from root `babelrc`, or just simply remove.

folders under `packages` should remove all `babel`, `eslint` ... settings. And keep it simple.

https://github.com/Canner/antd-canner-components/tree/master/packages/antd-array-tab_bottom

Reply:
It looks like in the latest babel, it does not automatically use the root `.babalrc` (gatsby also has .babelrc in their packages: https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-netlify) so I extend from root `babelrc`. Thanks.